gigantum/gigantum-client
# 1589 block import delete

## Pull Request for Issue #1589 

**Describe changes introduced by this PR (Bugs fixed, features added, docs updated, etc.)**:
Adds logic to block importing and displays message that importing is blocked because backup is in progress.


**Please check if the PR fulfills these requirements**

  - PR is set to merge to integration
  - PR title is descriptive, as this becomes the commit message on squash and merge.
  - Any related issues are added to the extended commit message during squash and merge.
  - Tests have been added (for bug fixes/features)
  - Docs have been added/updated (for bug fixes/features)

**QA Test Steps and Exit Criteria**:
Make sure ui is blocked and mutations fail when backup is in progress. Requires test server set into backup mode. Talk to @fmwest .
